internal/cli/cli.go

```diff
@@ -47,3 +51,65 @@ func (opts *globalOpts) OrgID() string {
 	opts.orgID = config.OrgID()
 	return opts.orgID
 }
+
+// deleteOpts options required when deleting a resource.
+// A command can embed this structure and then safely rely on the methods Confirm, DeleteFromProject or Delete
+// to manage the interactions with the user
+type deleteOpts struct {
```

Comment:
These are the main change and the rest is just deleting the duplicates to use this struct
We should be able now to embed this and reuse the Confirm, DeleteFromProject or Delete methods